stories/Subscription/SubscriptionSettings/index.stories.ts

```diff
@@ -0,0 +1,124 @@
+import type { Meta, StoryObj } from '@storybook/svelte'
+
+import Component from '@/ui/Pricing/SubscriptionSettings/index.svelte'
+
+const meta = {
+  // title: 'Design System/Icons',
+  component: Component,
+  // This component will have an automatically generated Autodocs entry: https://storybook.js.org/docs/react/writing-docs/autodocs
+  // tags: ['autodocs'],
+  parameters: {
+    // More on how to position stories at: https://storybook.js.org/docs/svelte/configure/story-layout
+    layout: 'fullscreen',
+
+    // docs: { story: { height: 100 } },
```

Comment:
It would be beneficial to have a single page with all states generated using `autodocs` tag. @vekkele could you please assign appropriate height for an iframe story (`docs: { story: { height: NUMBER } }`) in order for it to render correctly on docs page?